tests/integration_tests/performance/test_boottime.py

```diff
@@ -102,14 +120,28 @@ def test_boottime(
         vm.add_net_iface()
         vm.start()
         vm.pin_threads(0)
-        boottime_us = _get_microvm_boottime(vm)
-        metrics.put_metric("boot_time", boottime_us, unit="Microseconds")
-        timestamps = find_events(vm.log_data)
-        build_time = timestamps["build microvm for boot"]["duration"]
-        metrics.put_metric("build_time", build_time.microseconds, unit="Microseconds")
+
+        boot_time_us, cpu_boot_time_us = get_boottime_device_info(vm)
         metrics.put_metric(
             "guest_boot_time",
-            boottime_us - build_time.microseconds,
+            boot_time_us,
+            unit="Microseconds",
+        )
+        metrics.put_metric(
+            "guest_cpu_boot_time",
+            cpu_boot_time_us,
             unit="Microseconds",
         )
+
+        events = find_events(vm.log_data)
+        build_time = events["build microvm for boot"]["duration"]
+        metrics.put_metric("build_time", build_time.microseconds, unit="Microseconds")
+        resume_time = events["boot microvm"]["duration"]
+        metrics.put_metric("resume_time", resume_time.microseconds, unit="Microseconds")
+
+        kernel, userspace, total = get_systemd_analyze_times(vm)
+        metrics.put_metric("systemd_kernel", kernel, unit="Milliseconds")
+        metrics.put_metric("systemd_userspace", userspace, unit="Milliseconds")
+        metrics.put_metric("systemd_total", total, unit="Milliseconds")
```

Comment:
how stable are these wrt A/B-testing? (because they'll be automatically picked up in the A/B pipeline, so if these are volatile, we'll want to put them onto the ignorelist)

Reply:
I run A/B with last commit and it seems it is stable enough.

Reply:
Actually that was just one lucky run. `resume_time` appears to be more volatile than needed, so added it to ignore list.